third_party/blink/renderer/modules/mediasource/media_source.cc

```diff
@@ -178,6 +179,11 @@ SourceBuffer* MediaSource::addSourceBuffer(const String& type,
   if (!IsTypeSupportedInternal(
           GetExecutionContext(), type,
           false /* Allow underspecified codecs in |type| */)) {
+#if BUILDFLAG(IS_COBALT)
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+    LOG(INFO) << __func__ << "(" << type << ") is unsupported.";
```

Comment:
I don't think it's necessary as `MediaSource::addSourceBuffer()` is not to check the supportability of the mime type. It's more like a complimentary check here, so I think it's sufficient to only add extra log when it's rejected. Or, we can just add log in `IsTypeSupportedInternal()` instead of `isTypeSupported()` and `addSourceBuffer()`, though I think adding logs in `isTypeSupported()` and `addSourceBuffer()` would be more helpful.

Reply:
Discussed offline and added log for `MediaSource::addSourceBuffer()`.